grails-app/taglib/au/org/ala/specieslist/SpeciesListTagLib.groovy

```diff
@@ -58,4 +58,37 @@ class SpeciesListTagLib {
                 .replaceAll("\\(", "").replaceAll("\\)", "")
                 .toLowerCase()
     }
+
+
+    def selectedFacetLink = { attrs ->
```

Comment:
taglib methods are quite easy to unit test... might be worth adding some for these

Reply:
good feedback. ive added taglib tests.